utilities/blob_db/blob_file.h

```diff
@@ -34,6 +37,10 @@ class BlobFile {
   // after that
   uint64_t file_number_;
 
+  // If true, the keys in this file all has TTL. Otherwise all keys don't
+  // have TTL.
+  bool has_ttl_;
```

Comment:
Then this can be used to determine if a blob file is a simple blob-file or not, isn't it?

Reply:
yes.